src/plone/restapi/services/contextnavigation/get.py

```diff
@@ -251,7 +250,7 @@ def heading_link_target(self):
             # Go to the item /view we have chosen as root item
             return nav_root.absolute_url()
 
-    def root_type_name(self):
+    def root_type_title(self):
```

Comment:
@razvanMiu @nileshgulia1 can you elaborate on where this name change comes from?

Reply:
I guess this is a search/replace issue. I will revert that line. Let me know if that's ok with you.